.github/workflows/workspace.yml

```diff
@@ -0,0 +1,81 @@
+name: Workspace
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+
+permissions:
+  contents: read
+
+jobs:
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dtolnay/rust-toolchain@master
+      with:
+        toolchain: nightly-2024-06-11
+        components: clippy,rust-src
+    - name: Linux (linux_android_with_fallback.rs)
```

Comment:
Should we sort the steps in this job by filename? That might make it easier to tell that all the (non util) files are checked.

Reply:
I mostly followed the order used in lib.rs. I reordered it alphabetically, but it also may be worth to reorder (in a separate PR) lib.rs as well.